apps/health_campaign_field_worker_app/lib/pages/boundary_selection.dart

```diff
@@ -471,8 +471,10 @@ class _BoundarySelectionPageState
                                                   isClicked
                                               ? null
                                               : () async {
-                                                  if (!form.valid ||
-                                                      validateAllBoundarySelection()) {
+                                                  if (!form.valid
+                                                      // ||
+                                                      //     validateAllBoundarySelection()
+                                                      ) {
```

Comment:
I think validation should apply for first boundary.Please verify if it is as expected, remove commented code